addons/drag-drop/userscript.js

```diff
@@ -0,0 +1,42 @@
+export default async function ({ addon, global, console }) {
+  const DRAG_AREA_CLASS = "sa-drag-area";
+  const DRAG_OVER_CLASS = "sa-dragged-over";
```

Comment:
i think we're trying not to make our own classes

Reply:
See how it is used in the css... this is perfectly fine.